src/commons/sagas/RequestsSaga.ts

```diff
@@ -250,13 +250,13 @@ export const getAchievements = async (tokens: Tokens): Promise<AchievementItem[]
 };
 
 /**
- * GET /courses/{courseId}/admin/users/{studentCourseRegId}/goals
+ * GET /courses/{courseId}/staff/users/{studentCourseRegId}/goals
```

Comment:
Let's not do this. `/admin` doesn't refer to the role; it refers to (course) administration.

It's a sub-resources belonging to the REST resource which is the course itself.

The role check is transparent (i.e. unknown to) the frontend; you shouldn't be able to tell based on the endpoint what are the checks.